apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -1087,6 +1088,34 @@ function handleAddPost( calypsoPort ) {
 	handleWpAdminRedirect( { calypsoPort, path: 'post-new.php', title: __( 'Add new post' ) } );
 }
 
+async function handleWpTemplateCommands( calypsoPort ) {
+	const callback = ( e, postType, postId ) => {
+		e.preventDefault();
+
+		calypsoPort.postMessage( {
+			action: 'wpAdminRedirect',
+			payload: {
+				destinationUrl: `/wp-admin/site-editor.php?postType=${ postType }&postId=${ encodeURIComponent(
+					postId
+				) }&canvas=edit`,
+				unsavedChanges: select( 'core/editor' ).isEditedPostDirty(),
+			},
+		} );
+	};
+
+	const entityNames = [ 'wp_template', 'wp_template_part' ];
+
+	entityNames.forEach( async ( entityName ) => {
+		const records = await getPostTypeRecords( entityName );
+
+		records.forEach( ( record ) => {
+			const selector = `[data-value$="${ record.id }"]`;
```

Comment:
What do you think about making this selector a bit more specific like `.commands-command-menu__container [data-value$="${ record.id }"]`? Would it improve the performance?

Reply:
Since the selector ends up being used in a `closest` call, the data attribute selector is more performant, since it's simpler for the browser to match against. As ChatGPT puts it:

```
The first selector ([data-value$="${record.id}"]) is faster because it's a simpler operation, only needing to find an ancestor element matching a single attribute selector.
The second selector (.commands-command-menu__container [data-value$="${record.id}"]) is slower due to the additional specificity of having to find an element by both class and attribute, plus the overhead of checking descendants.
```